src/vl-map-draw-point-action.js

```diff
@@ -15,7 +15,27 @@ import {VlDrawAction, OlGeometryType} from '/node_modules/vl-mapactions/dist/vl-
  */
 export class VlMapDrawPointAction extends VlMapDrawAction {
```

Comment:
Als we deze uitbreiding op VlMapDrawAction voorzien, werkt dit dan ook niet al out of the box met `VlMapDrawLineAction` en `VlMapDrawPolygonAction`? Enige dat dan nog voorzien moet worden is e2e testen?

Reply:
Uitgebreid + op unit test niveau meegenomen.